CoreScriptsRoot/Modules/Settings/Pages/GameSettings.lua

```diff
@@ -103,10 +103,38 @@ local function Initialize()
 
 		settingsDisabledInVR[this.FullscreenEnabler] = true
 
-		local fullScreenSelectionFrame = this.FullscreenEnabler.SliderFrame and this.FullscreenEnabler.SliderFrame or this.FullscreenEnabler.SelectorFrame
-
+		local Debounce = false
```

Comment:
The debounce was from another angle I was trying to fix the bug from. I guess I never tested this way without the debounce, seems to work fine without it. I'll remove it.